test/integration/targets/ios_acl/tests/cli/deleted.yaml

```diff
@@ -0,0 +1,68 @@
+---
+- debug:
+    msg: "Start Deleted integration state for ios_acl ansible_connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate_config.yaml
+
+- block:
+    - name: Delete attributes of provided configured interfaces
+      ios_acl: &deleted
+        config:
+          - afi: ipv4
+            acls:
+              - name: test_acl
+                acl_type: extended
+              - name: 110
+              - name: 123
+          - afi: ipv6
+            acls:
+            - name: R1_TRAFFIC
+        state: deleted
+      register: result
+
+    - assert:
+        that:
+          - "result.commands|length == 4"
+          - "result.changed == true"
+          - "result.commands|symmetric_difference(deleted.commands) == []"
+
+    - name: Delete attributes of all configured interfaces (IDEMPOTENT)
+      ios_acl: *deleted
+      register: result
+
+    - name: Assert that the previous task was idempotent
+      assert:
+        that:
+          - "result.commands|length == 0"
+          - "result.changed == false"
+
+    - include_tasks: _remove_config.yaml
+    - include_tasks: _populate_config.yaml
+
+    - name: Delete ACL attributes of provided configured interfaces based on AFI
+      ios_acl: &deleted_afi
+        config:
+          - afi: ipv4
+        state: deleted
+      register: result
+
```

Comment:
Can you please add a test for deleting a single ACE?

Reply:
+1